tests/imposter/private-api-config.yaml

```diff
@@ -146,3 +190,10 @@ resources:
             "error": "server_error"
           }
         }
+
+  - method: POST
+    path: /abandon
+    response:
+      statusCode: 200
+      content: |
+        {}
```

Comment:
I see we have a unit test for the failure scenario for this API call, do we want a browser test too?

Reply:
Resolving as this has now been addressed 🎉